java11/pom.xml

```diff
@@ -29,12 +39,38 @@
                <release>11</release>
             </configuration>
          </plugin>
+
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Why are you shading the dependency ?

Reply:
I don't want bytebuddy to escape. It's toxic.